src/account/EphemeralKeyPair.ts

```diff
@@ -0,0 +1,157 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { randomBytes } from "@noble/hashes/utils";
+
+import {
+  EPK_HORIZON_SECS,
+  Ed25519PrivateKey,
+  EphemeralPublicKey,
+  EphemeralSignature,
+  PrivateKey,
+} from "../core/crypto";
+import { Hex } from "../core/hex";
+import { bytesToBigIntLE, padAndPackBytesWithLen, poseidonHash } from "../core/crypto/poseidon";
+import { EphemeralPublicKeyVariant, HexInput } from "../types";
+import { Deserializer, Serializable, Serializer } from "../bcs";
+import { currentTimeInSeconds, floorToWholeHour } from "../utils/helpers";
+
+export class EphemeralKeyPair extends Serializable {
```

Comment:
do we have any live docs we can link here as string docs? also, let's add string doc to this class so it would show up nicely on the developer ide